packages/discovery-react-components/src/components/SearchFacets/utils/searchFilterTransform.tsx

```diff
@@ -79,50 +65,38 @@ export class SearchFilterTransform {
     };
   }
 
-  // Returns a query string by combining field and dynamic query strings
   static toString(facets: SearchFilterFacets): string {
     const fieldFilters = this.fieldsToString(facets.filterFields);
-    const dynamicFilters = this.escapeSelectedFacets(facets.filterDynamic, 'text').join(',');
+    const dynamicFilters = this.quoteSelectedFacets(facets.filterDynamic, 'text').join(',');
     return [fieldFilters, dynamicFilters].filter(Boolean).join(',');
   }
 
-  private static unescapeString(escapedString: string): string {
-    // Remove the parentheses added by escapeSelectedFacets
-    // Unescape the special characters within the string
-    const unescapedString = escapedString
-      .replace(RegExp(SearchFilterTransform.STRING_IN_PARENS), '$1')
-      .replace(RegExp(SearchFilterTransform.ESCAPED_CHAR), '$1');
-    return unescapedString;
+  private static unquoteString(quotedString: string): string {
+    return quotedString.replace(/^"(.+)"$/, '$1').replace(/\\"/, '"');
   }
 
-  // Returns a full filter query string from the set of filterFields
-  // Use the includes query operator (:) for all fields
-  // @see https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-query-operators#includes
   static fieldsToString(facets: InternalQueryTermAggregation[]): string {
     const filterStrings: string[] = [];
     facets.forEach(facet => {
       const field = get(facet, 'field', '');
       const results = get(facet, 'results', []);
-      const keys = this.escapeSelectedFacets(results, 'key');
+      const keys = this.quoteSelectedFacets(results, 'key');
       if (keys.length) {
-        filterStrings.push(keys.map(key => `${escapeFieldName(field)}:${key}`).join('|'));
+        filterStrings.push(keys.map(key => `${escapeFieldName(field)}::${key}`).join('|'));
```

Comment:
Same as before: add comment to function, stating that it is using double-colon and what effect(s) that has.